readme/Readme.scalatex

```diff
@@ -1003,6 +1052,23 @@
             @p
                 These operations are not cheap: the bitsets easily take a few KB of memory each, and involve 65k iterations to fill them in. However, since @hl.scala{Parser}s are immutable, this one-time-cost goes from "ridiculous" to "acceptable". All these internal optimizations are completely opaque to the user, who (apart from performance) never need to think about them.
 
+        @sect{Parsing Only}
+            @p
+                Fastparse, unlike libraries like @lnk("Scodec", "https://github.com/scodec/scodec")
+                or Python's @lnk("Construct", "https://pypi.python.org/pypi/construct"),
+                only allows you to define how to parse data from serialized
+                text/bytes into structured data, and not the other way around.
+
+            @p
+                It is probably possible to write a wrapper on top of FastParse
+                that will allow two-directional serialization, but such a thing
```

Comment:
Btw, can you explain what "two-directional serialization" means? :) I'm just curious.